composer.json

```diff
@@ -223,12 +222,11 @@
         "npm-asset/tablesaw": "~3.1.0",
         "oomphinc/composer-installers-extender": "~1.0 || ~2.0",
         "spatie/color": "^1.2",
-        "swiftmailer/swiftmailer": "6.2.4",
```

Comment:
Ronald I added to the commit message https://github.com/goalgorilla/open_social/pull/2919/commits/6280273a70c95d38e47a69038fed4b8838ac9a07 maybe we should add to another channel?

Reply:
Basically we should check if we should add it to a different channel, we used to lock the version to `"swiftmailer/swiftmailer": "6.2.4"`
however, by removing the dependency here, we are now using the dependency from 
https://git.drupalcode.org/project/swiftmailer/-/blob/8.x-2.x/composer.json#L11

Which isn't locking the version, so it's likely it will resolve to a different version now
Your commit message clearly explains why, but not what version we now upgrade/downgraded to.

Reply:
Done, added as a change record.